webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+
+  module: {
```

Comment:
- [ ] Nice work using Webpack for the implementation of the project :+1:  However Webpack **dev server** is not set up in your Webpack configuration. Kindly make sure Webpack **dev server** is set up and running with no errors